packages/shared/src/components/tabs/TabList.tsx

```diff
@@ -126,13 +131,14 @@ function TabList<T extends string = string>({
               className.item,
               'relative p-2 py-4 text-center font-bold typo-callout',
               isActive ? '' : 'text-text-tertiary',
+              isAnchor && 'cursor-pointer',
```

Comment:
Non mandatory: if we have more conditional classes maybe the object structure can be more readable, wdyt?

```ts
classNames(
              className.item,
              'relative p-2 py-4 text-center font-bold typo-callout',
              { 'text-text-tertiary': !isActive, 'cursor-pointer': isAnchor},
)
```

Reply:
Don't think we'll have any more now